src/taipy/gui/renderers/builder.py

```diff
@@ -717,9 +718,15 @@ def set_value_and_default(
             default_val (optional(Any)): the default value.
         """
         var_name = self.__default_property_name if var_name is None else var_name
-        if var_type == PropertyType.number_or_lov_value:
-            var_type = PropertyType.lov_value if self.__attributes.get("lov") else PropertyType.dynamic_number
-            native_type = native_type if var_type == PropertyType.dynamic_number else False
+        if var_type == PropertyType.slider_value:
+            if self.__attributes.get("lov"):
+                var_type = PropertyType.lov_value
+                native_type = False
+            else:
+                var_type = (PropertyType.dynamic_lo_numbers
+                            if isinstance(self._Builder__attributes.get("value"), collections.abc.Sequence)
```

Comment:
ouch self._Builder__attributes => self.__attributes

Reply:
Well... if I do that I get "AttributeError: '_Builder' object has no attribute '__attributes'"...
What gives?

Reply:
iut is used line 722 ...

Reply:
Works fine after all... Thx